azure-iot-device/azure/iot/device/provisioning/internal/polling_machine.py

```diff
@@ -9,7 +9,7 @@
 import traceback
 from threading import Timer
 from transitions import Machine
-from ..transport import constant
+from ..pipeline import constant
```

Comment:
>.. [](start = 5, length = 2)

relative imports that involve going up a directory make me queasy.

If you have to go up a level, you should probably use the full import namespace instead